### ISSUE TITLE
`uninstall` doesn't work after an upgrade

### ISSUE THREAD

**Issue by LukeShu** [telepresenceio/telepresence#1666]
We've been getting occasional reports about things like this:
```console
$ telepresence uninstall --everything
telepresence: error: multiple errors:
  unable to locate port rest in container system-a in Deployment system-a.saas-beta
  unable to locate port http in container sso in Deployment sso.saas-beta
  unable to locate port http in container system-a-ui in Deployment system-a-ui.saas-beta
  ```
and so far we've been saying "something else must have modified the service under Telepresence's nose".

And some of the time, that probably is the case, but then Peter reported after upgrading to 2.1.5:
```console
$ telepresence uninstall --everything
telepresence: error: multiple errors:
  object does not contain a "traffic-annotations" volume
  object does not contain a "traffic-annotations" volume
  object does not contain a "traffic-annotations" volume
```

(Besides that the error message doesn't include object names -_-), this reveals an underlying issue: That it's ignoring the annotation telling is which telepresence version installed the thing, so that we can do version-specific cleanup.  It's always just doing the most recent version of the cleanup!

**Comment by LukeShu** [comment]
Probably a dup of https://github.com/telepresenceio/telepresence/issues/1658

**Comment by khussey** [comment]
This is fixed in the 2.2.0 release, which is now available.
